scene/resources/atlas_texture.cpp

```diff
@@ -94,6 +94,7 @@ void AtlasTexture::set_region(const Rect2 &p_region) {
 		return;
 	}
 	region = p_region;
+	rounded_region = Rect2(p_region.position, p_region.size.round());
```

Comment:
Looks like this one should be `floor()` instead on `round()` (recently I tried to take into account margins while remapping uv in `draw_polygon` and failed, but at least `floor()` usage was correct).

Reply:
I changed it to `floor()`. Now it seems consistent with margins, which are rounded implicitly.

Reply:
Thanks!
(But I'm not actually sure that margins rounded implicitly)

Reply:
Sorry, I mean not sure they are rounded implicitly when calling `get_region_rect()` which is the problem.

Reply:
I tested again and this implicit margin casting is not actually _margin_ casting. It is casting of region + margin (set region.w=1.5 and margin.w=1.49 and then slowly increase margin.w above 1.5).

Reply:
So does anything need to be changed?

Reply:
`margins` can be freely left unchanged.
Thanks!